hugo.yaml

```
@@ -237,9 +237,9 @@ params:
       enable: true
       # The responses that the user sees after clicking "yes" (the page was helpful) or "no" (the page was not helpful).
       'yes': >-
-        Glad to hear it! If you have further feedback, please <a href="https://github.com/wandb/docs/issues/new?template=praise.md">let us know</a>.
+        Glad to hear it! If you have more to say, please <a href="javascript:location.href='https://github.com/wandb/docs/issues/new?template=praise.md&title=Praise:%20' + encodeURIComponent(document.title) + '&body=Page:%20' + encodeURIComponent(location.href)">let us know</a>.
```

Nearly there. We need to remove the substring ` | Weights & Biases Documentation` from the `document.title`. I tried doing it like this, but it is causing a syntax error:
```suggestion
        Glad to hear it! If you have more to say, please <a href="javascript:location.href='https://github.com/wandb/docs/issues/new?template=praise.md&title=Praise:%20' + encodeURIComponent(document.title).replaceAll(' | Weights & Biases Documentation','') + '&body=Page:%20' + encodeURIComponent(location.href)">let us know</a>.
```